modules/decklinksrc/decklinkSrcWorker.py

```diff
@@ -9,7 +9,8 @@ def decklinkSrcWorker(pause, exitRequest, pipelineOut):
         pause.release()
 
         curr_frame = decklinkSrc.grab()
-        pipelineOut.put(curr_frame)
+        if curr_frame is not None:
+            pipelineOut.put(curr_frame)
 
 
         if not exitRequest.empty():
```

Comment:
Add a comment. What does non-empty exitRequest mean? What is exitRequest at all?

Reply:
exitRequest is the exit pipeline. If its not empty, that means something was put into it and a stop order was issued.